lib/test/t_FunctionalChaos_2d.expout

```diff
@@ -0,0 +1,14 @@
+ProjectionStrategy (repr)= 
+class=ProjectionStrategy implementation=class=LeastSquaresStrategy experiment=class=FixedExperiment name=Unnamed sample=class=Sample name=ComposedDistribution implementation=class=SampleImplementation name=ComposedDistribution size=100 dimension=2 description=[X0,X1] data=[[0.608202,-1.51696],[-1.26617,-1.29938],[-0.438266,0.230372],[1.20548,-3.09737],[-2.18139,0.01323],[0.350042,-1.25743],[-0.355007,1.02776],[1.43725,-0.766431],[0.810668,0.217512],[0.793156,1.04533],[-0.470526,0.331569],[0.261018,-0.488205],[-2.29006,-0.465482],[-1.28289,0.332084],[-1.31178,-0.167726],[-0.0907838,3.01263],[0.995793,0.94204],[-0.139453,0.61189],[-0.560206,0.611715],[0.44549,-1.5375],[0.322925,-2.4067],[0.445785,0.662936],[-1.03808,-0.65616],[-0.856712,-0.751611],[0.473617,0.438177],[-0.125498,-0.455335],[0.351418,1.86038],[1.78236,0.219721],[0.0702074,1.72546],[-0.781366,-0.543405],[-0.721533,-0.736749],[-0.241223,-0.508206],[-1.78796,-2.25867],[0.40136,-0.5964],[1.36783,-0.31468],[1.00434,-1.78274],[0.741548,-0.684734],[-0.0436123,0.0611157],[0.539345,0.87372],[0.29995,-1.46295],[0.407717,-0.318786],[-0.485112,1.26314],[-0.382992,-0.426726],[-0.752817,-1.89234],[0.257926,-0.514391],[1.96876,0.647229],[-0.671291,0.00370249],[1.85579,0.729688],[0.0521593,-0.247234],[0.790446,0.479191],[0.716353,-0.0336098],[-0.743622,-0.0367271],[0.184356,0.110256],[-1.53073,-0.37687],[0.655027,-0.0955894],[0.538071,0.109122],[1.73821,-0.198754],[-0.958722,0.47362],[0.377922,0.161637],[-0.181004,0.384483],[1.67297,0.116468],[-1.03896,-0.10008],[-0.353552,1.49156],[1.21381,1.22301],[-0.777033,0.526646],[-1.36853,-0.656923],[0.103474,-0.131228],[-0.89182,-1.45347],[0.905602,1.17414],[0.334794,0.929395],[-0.483642,-0.337113],[0.677958,0.578688],[1.70938,-0.582459],[1.07062,-1.38886],[-0.506925,-0.499748],[-1.66086,-1.55516],[2.24623,0.483083],[0.759602,0.205004],[-0.510764,-0.0972525],[-0.633066,0.592563],[-0.957072,-0.602044],[0.544047,-1.21009],[0.814561,-0.886698],[-0.734708,-0.141114],[-0.111461,0.441983],[0.994482,0.519162],[-0.160625,-1.51455],[-0.938771,-0.676917],[-1.96869,0.667678],[-0.657603,-1.40585],[0.338751,-0.0295335],[1.01556,-0.631829],[0.637167,-0.342157],[-0.0899071,2.05339],[-0.855886,1.1587],[1.27128,-1.45717],[-0.238253,-0.844367],[1.3263,-0.28861],[2.11968,0.419271],[-0.901581,-0.836064]] weights=class=Point name=Unnamed dimension=100 values=[0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01,0.01]
```

Comment:
no expout files for new tests please

Reply:
When a specific unit test is producing numerical values which are tested using `assert` statements, I agree that `expout` files are unwanted. In this particular case, the expected output is the content of the `repr` and `str` methods. How can we make sure that the actual content is the expected one without checking it? More precisely, one downside of removing the `expout` file here is that the pretty-print feature may be broken and no test will ever fail so that the missing feature may go unnoticed.